src/components/Menu/README.md

```diff
@@ -0,0 +1,319 @@
+<!--GITHUB_BLOCK-->
+
+# Menu
+
+<!--/GITHUB_BLOCK-->
+
+```tsx
+import {Menu} from '@gravity-ui/uikit';
+```
+
+## Description
+
+The Menu component makes it easy to navigate the sections of a website or other entity for which this menu is used. With the help of the menu, the user can easily navigate to the sections of interest, understand what information is presented in essence, without studying it completely.
+
+The Menu component also provides Menu components for Menu.Item and Menu.Group. With them, you can easily build your menus by combining different options
+
+<!--LANDING_BLOCK
+<ExampleBlock
+    code={`
+<Menu>
+    <Menu.Item>First</Menu.Item>
+    <Menu.Item>Second</Menu.Item>
+</Menu>
+`}
+>
+    <UIKit.Menu>
+        <UIKit.Menu.Item>First</UIKit.Menu.Item>
+        <UIKit.Menu.Item>Second</UIKit.Menu.Item>
+    </UIKit.Menu>
+</ExampleBlock>
+LANDING_BLOCK-->
+
+<!--GITHUB_BLOCK-->
+
+```tsx
+<Menu>
+  <Menu.Item>First</Menu.Item>
+  <Menu.Item>Second</Menu.Item>
+</Menu>
+```
+
+<!--/GITHUB_BLOCK-->
+
+### Size
+
+Used to select the menu size: `s | m | l | xl`, default - `m`
+
+<!--LANDING_BLOCK
+<ExampleBlock
+    code={`
+<Menu size="l">
+    <Menu.Item>First</Menu.Item>
+    <Menu.Item>Second</Menu.Item>
+</Menu>
+`}
+>
+    <UIKit.Menu size="l">
+        <UIKit.Menu.Item>First</UIKit.Menu.Item>
+        <UIKit.Menu.Item>Second</UIKit.Menu.Item>
+    </UIKit.Menu>
+</ExampleBlock>
+LANDING_BLOCK-->
+
+<!--GITHUB_BLOCK-->
+
+```tsx
+<Menu size="l">
+  <Menu.Item>First</Menu.Item>
+  <Menu.Item>Second</Menu.Item>
+</Menu>
+```
+
+<!--/GITHUB_BLOCK-->
+
+## Properties
+
+| Name          | Description                             | Default |
+| :------------ | :-------------------------------------- | :-----: |
+| **size**      | Menu size `s - m - l - xl`              |    m    |
+| **children**  | Child element `React.ReactNode`         |    -    |
+| **className** | HTML `class` attribute `string`         |    -    |
+| **style**     | HTML `style` attribute `CSSProperties`  |    -    |
+| **qa**        | HTML `data-qa` attribute, used in tests |    -    |
+
+# Menu.Item
```

Comment:
Level 1 header must be only one on a page. Let's describe item and group in main sections